docs/source/challenge-doc.rst

```diff
@@ -110,10 +119,29 @@ Observation Space
 
     - Hand passing error measures the distance between the MPL and the object
 
-    - The manipulated object has full 6 degrees of freedom, its state described as a 7 dimensional value in position + quaternion format  it is defined by a "`freejoint <https://mujoco.readthedocs.io/en/stable/XMLreference.html#body-freejoint>`__"
+    - The manipulated object has full 6 degrees of freedom, its state described as a 7 dimensional value in position + quaternion format. Details can be found in "`freejoint <https://mujoco.readthedocs.io/en/stable/XMLreference.html#body-freejoint>`__"
+
+
+
+
+**Variation on Object Properties**
+The geometry, mass, and friction of the object will reset at the start of each episode. 
 
+    - Object scale: a +- change between 0% - 5%, 0% - 10% scale variations in respective geom directions 
+    - Object Mass: an upper/lower bound of +-50 gms
+    - Object Friction: a +- change between 0 - 0.1, 0 - 0.001, 0 - 0.00002 from nominal value: [1.0, 0.005, 0.0001] in respective geom direction
 
+**Success Condition**
 
+    - The object moved from start position to goal position. Both the MPL hand, and MyoHand, is required to touch the object for 100 timesteps 
+    - Exerting a maximum contact force on the object, less than 1500N (subject to change based on submission)
+    - Placing the object within 0.05 meters of the goal site on the pillar
+
+**Ranking Criteria**
+    1. Task success rate (success_attempt / total_attempt)
+    2. Time to complete the task (success_attempt + failed_attempt)
+    3. Muscle activation
```

Comment:
total / average?

Reply:
Updated to "total" for all three mentioned in [commit](https://github.com/MyoHub/myosuite/commit/e31145ea1a059add2cb32370ebb6951a8b3071a7)

Reply:
This will need more thought and clarification @Balint-H @cherylwang20 
- If someone fails in the first step they will have the lowest `Minimum total muscle activation`
- If the idea is that we will only make sense conditioned on the two solutions having the same success, it needs to be written that way 

and similarly for other points.

Reply:
Include better description on the hierarchy of evaluation. updated in [commit](https://github.com/MyoHub/myosuite/commit/d192bd93b0f94754cf9a9f4940b77a6e0c21c29a)